src/locale/locales/ca/messages.po

```diff
@@ -929,7 +929,7 @@ msgstr "Configura els filtres de continguts per la categoria: {0}"
 
 #: src/components/moderation/LabelPreference.tsx:81
 msgid "Configure content filtering setting for category: {name}"
-msgstr ""
+msgstr "Configura els filtres de continguts per la categoria: {nom}"
```

Comment:
I noticed one small thing here, the variable (or whatever the correct name is) should not be translated, so it should be `{name}` at the end rather than `{nom}`.

Also, well done on your efforts at keeping the translation so up-to-date :)

Reply:
nice, thanks